README.md

```diff
@@ -52,10 +52,66 @@ Considering the very rich and diverse flora of climate indices this
 is however not always possible. By collecting a wide range of
 different indices it is easier to discover any common patterns and
 features that are currently not well covered by the CF Conventions.
-Currently identified issues frequently relate to
-[`standard_name`](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.8/cf-conventions.html#standard-name)
-or/and [`cell_methods`](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.8/cf-conventions.html#cell-methods)
-which both are *controlled vocabularies* of the CF Conventions.
+
+With respect to climate index metadata the CF Conventions are in a sense
+both *permissive* and *restrictive*. This have have some bearing on which indices
+are at all incuded in the list, and which metadata elements are present,
+as well as the (somewhat subjective) classification as `Ready` or not.
+
+<details>
+<p><summary><b><u>Further details here <i>(click here)</i></u></b></summary></p>
+
+CF is *permissive* in the sense that only the bare essential information for
+understanding what the data represents are mandatory, and that any additional
+information can be included with few limitations. Thus, almost any climate
+index dataset can be published in a CF compliant way, but only with the bare
+minimum of standardised metadata. For such free (i.e., non-managed) information
+there are no rules, which means that it is difficult or intractable to develop
+common standardised workflows that would depend on this particular metadata
+information. For such purposes CF is *restrictive* in that the managed
+components are not always well suited to handle climate index metadata.
+
+In the *index definition* table there is a column *ready* (second left) that is
+subjectively indicating how complete the metadata description is. In general
+terms, indices marked as ready ("1") either have full metadata description,
+or there are advanced plans for what needs to be done. In particular, the focus
+is on the following elements of the CF Conventions:
+
+*  [`standard_name`](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#standard-name) (recommended if available).
+
+*  [`long_name`] (free text, recommended).
```

Comment:
Missing link ? Otherwise the brackets seems unnecessary.
Same goes for unit below.